qiskit/visualization/circuit/_utils.py

```diff
@@ -119,10 +118,11 @@ def get_gate_ctrl_text(op, drawer, style=None, calibrations=None):
 
 def get_param_str(op, drawer, ndigits=3):
     """Get the params as a string to add to the gate text display"""
-    if not hasattr(op, "params") or any(isinstance(param, np.ndarray) for param in op.params):
-        return ""
-
-    if isinstance(op, ControlFlowOp):
+    if (
+        not hasattr(op, "params")
+        or any(isinstance(param, np.ndarray) for param in op.params)
+        or any(isinstance(param, QuantumCircuit) for param in op.params)
+    ):
```

Comment:
This looks fine. It would be good to add tests for these 3 cases. Though the code is in `utils.py`, the easiest place to test it would probably be in `test_circuit_text_drawer`. Also a simple release note indicating that a `QuantumCircuit` as a parameter will no longer display in the circuit drawers due to formatting issues.

Reply:
Thanks so much @enavarro51. I wasn't sure if additional tests were needed since this code was a minor change for something I thought was already being tested. 

Also, thanks for pointing out where the tests should go. I will look into `test_circuit_text_drawer` and add them. Just to double check, are tests only the text drawer enough? Or do other drawers (like mpl) need them too?

I'll add the reno too.

Reply:
Yeah, there apparently aren't any tests for these 3 options. The text drawer is sufficient. You're really just checking the call to `_get_param_str` returns "". You could do that in `test_utils` if you want, but I think it's nice to have a visual output for visualizations.